README.arm.md

```diff
@@ -68,16 +68,23 @@ ARMv6 as described in the Build Dependencies section above.
 
 ## Raspberry Pi 2
 
-For Raspberry Pi 2, which is ARMv7, the default build should work. In
-the case of Raspberry Pi 2, in case the default build fails, download
-LLVM binaries from the LLVM website.
+The Raspberry Pi 2 ARM CPU type is also not detected by LLVM.
+Similar to the above instructions for the Raspberry Pi, define the
```

Comment:
Okay, going to adjust the Raspberry Pi section too then (it advises to `export` the variable, which seems fragile to me).